### DISCUSSION TITLE
BGM for different parts of the round

### DISCUSSION THREAD

**Discussion by Amara-MFG** [ikemen-engine/Ikemen-GO#1997] [Feature Request] [answered]
### Is your feature request related to a problem? Please describe.

It would also be nice if we could toggle the option on fight.def to make it so the stage music starts after the "Fight!" signal (see KOF XI for example or KOF 95).

### Describe the solution you'd like

A way to add a stage song or intro song before the round gets announced or during character intros and a BGM for when the round starts after the "FIGHT!" signal.

It would also be nice if we could toggle this option on fight.def

See KOF95, SFA2 (super nintendo) and KOF XI for examples.

**Comment by jankespro12** [comment]
Does KOF2003 also count too?

**Reply by Amara-MFG** [reply]
If you want to, yes

**Comment by Jesuszilla** [comment, marked as answer]
this is already possible in nightly; use playBGM SCTRL and use "stage" as the argument to bgm to recall the stage BGM that was playing. Use BGMVar to retrieve the original values of loop points, start position, etc. of the stage BGM before calling playBGM.

**Reply by jankespro12** [reply]
so uh, can you teach me how? because i wanted to recreate the KOF2003's Stage Intro effect.
